upgrading/automated_upgrades.adoc

```diff
@@ -20,300 +20,159 @@ endif::[]
-----
-
-Whenever the *_openshift_node_group.yml_* playbook is run, the changes defined
-in the `edits` field will update the related ConfigMap (`node-config-compute` in
-this example), which will ultimately affect the node's configuration file on the
-host.
+- Upgrade existing {product-title} version 3.10 clusters to version 3.11.
+- Upgrade {product-title} version 3.11 clusters to the latest
+xref:../release_notes/ocp_3_11_release_notes.adoc#ocp-311-asynchronous-errata-updates[asynchronous
+errata update].
 
 [[upgrade-workflow]]
-=== Upgrade Workflow
+== Upgrade workflow
 
-The 3.9 to 3.10 control plane upgrade performs the following steps for
+The 3.10 to 3.11 control plane upgrade performs the following steps for
 you:
 
-* A backup of all etcd data is taken for recovery purposes.
-* The API and controllers are updated from 3.9 to 3.10.
-* Internal data structures are updated to 3.10.
-* The default router, if one exists, is updated from 3.9 to 3.10.
-* The default registry, if one exists, is updated from 3.9 to 3.10.
-* The default image streams and InstantApp templates are updated.
+* Back up all etcd data for recovery purposes.
+* Update the API and controllers from 3.10 to 3.11.
+* Update internal data structures to 3.11.
+* Update the default router, if one exists, from 3.10 to 3.11.
+* Update the default registry, if one exists, from 3.10 to 3.11.
+* Update the default image streams and InstantApp templates.
 
-The 3.9 to 3.10 node upgrade performs a rolling update of nodes, which:
+The 3.10 to 3.11 node upgrade performs a rolling update of nodes, which:
 
 * Marks a subset of nodes unschedulable and drains them of pods.
-* Updates node components from 3.9 to 3.10.
-* Converts from local configuration to bootstrapped TLS and Config.
-* Converts SDN components from *systemd* services to DaemonSets.
+* Updates node components from 3.10 to 3.11.
 * Returns those nodes to service.
 
-[IMPORTANT]
-====
-* Ensure that you have met all
-xref:../install/prerequisites.adoc#install-config-install-prerequisites[prerequisites]
-before proceeding with an upgrade. Failure to do so can result in a failed
-upgrade.
+[[upgrade-prerequisites]]
+== Prerequisites
+
+Before you upgrade your cluster:
 
-* If you are using GlusterFS, see
-xref:special-considerations-for-glusterfs[Special Considerations When Using
-Containerized GlusterFS] before proceeding.
+* Review the
+xref:../release_notes/ocp_3_11_release_notes.adoc#release-notes-ocp-3-11-release-notes[{product-title} 3.11 Release Notes].
+The release notes contain important notices about changes to {product-title} and
+its function.
 
-* If you are using GCE Persistent Disk (gcePD), see xref:special-considerations-for-gcepd[Special Considerations When Using
-gcePD] before proceeding.
+* Upgrade the cluster to the
```

Comment:
@kalexand-rh this section needs updating for 3.11.

Reply:
@vikram-rh, can you please be more specific? I thought you needed to upgrade to the latest version of 3.10 before you upgraded to 3.11.